nautobot/dcim/models/devices.py

```diff
@@ -408,12 +430,65 @@ class Platform(OrganizationalModel):
     )
     description = models.CharField(max_length=200, blank=True)
 
+    @cached_property
+    def _network_drivers(self):
+        NETWORK_DRIVERS = deepcopy(get_settings_or_config("NETWORK_DRIVERS"))
+        for key, mapping in (
+            ("ansible", ANSIBLE_LIB_MAPPER_REVERSE),
+            # TODO ("hier_config", HIER_CONFIG_LIB_MAPPER_REVERSE),
+            ("netmiko", NETMIKO_LIB_MAPPER_REVERSE),
+            ("ntc_templates", NTCTEMPLATES_LIB_MAPPER_REVERSE),
+            ("pyats", PYATS_LIB_MAPPER_REVERSE),
+            ("pyntc", PYNTC_LIB_MAPPER_REVERSE),
+            ("scrapli", SCRAPLI_LIB_MAPPER_REVERSE),
+        ):
+            # Fill in values from *LIB_MAPPER_REVERSE but let NETWORK_DRIVERS override them
+            NETWORK_DRIVERS[key] = {**mapping, **(NETWORK_DRIVERS.get(key, {}))}
+        return NETWORK_DRIVERS
+
+    @property
+    def ansible_driver(self):
+        """Collection string for use with Ansible; derived from network_driver by netutils."""
+        return self._network_drivers["ansible"].get(self.network_driver, None)
```

Comment:
Are the `None`'s required?

Reply:
Don't think so, but I figured it didn't hurt to be explicit about the default behavior.